src/components/attribute/EmptyState.tsx

```diff
@@ -0,0 +1,173 @@
+const ICON = aha.iconForExtensionIdentifier(IDENTIFIER);
+
+type MenuProps = {
+  record: LinkableRecord;
+  onPaste: Function;
+};
+
+const Menu = ({ record, onPaste }: MenuProps) => {
+  const handleSync = () => {
+    aha.command(`${IDENTIFIER}.sync`, { record });
+  };
+
+  return (
+    <aha-menu>
+      <aha-button slot="button" type="attribute" size="mini">
+        <aha-icon icon="fa-solid fa-ellipsis"></aha-icon>
+      </aha-button>
+      <aha-menu-item onClick={() => onPaste()}>Paste PR link</aha-menu-item>
+      <aha-menu-item onClick={handleSync}>Scan GitHub</aha-menu-item>
+      <hr />
+      <aha-menu-item>
+        <a href={LEARN_MORE_URL} target="_blank" rel="noopener noreferrer">
+          <aha-icon icon="fa fa-external-link" />
+          Read the docs
+        </a>
+      </aha-menu-item>
+    </aha-menu>
+  );
+};
+
+const EmptyStateBox = ({ children }) => (
+  <aha-box class="m-0" style={{ color: "var(--theme-secondary-text)" }}>
+    <div style={{ margin: "calc(-2em + 12px)" }}>{children}</div>
+  </aha-box>
+);
+
+export const EmptyState: React.FC<{ record: LinkableRecord }> = ({
+  record,
+}) => {
+  const [onCopy, copied] = useClipboard();
+  const [pasteMode, setPasteMode] = useState<Boolean>(false);
+  const [validation, setValidation] = useState<String | null>(null);
+  const [hasConfiguredWebhook, setHasConfiguredWebhook] =
+    useState<Boolean | null>(null);
+
+  // aha-menu gets mad if you remove it from the DOM while the menu is open.
+  // Wait 1 tick so the menu has a chance to unmount first.
+  const viewPasteMode = () => {
+    setTimeout(() => setPasteMode(true), 1);
+  };
+
+  const pasteLink = async (url: string) => {
+    if (!validPrUrl(url)) {
+      setValidation("Please enter a valid pull request URL");
+      return;
+    }
+
+    await withGitHubApi(async (api) => {
+      const pullRequest = await getPrByUrl(api, url);
+      await updatePullRequestLinkOnRecord(pullRequest, record);
+      setPasteMode(false);
+    });
+  };
+
+  // Song and dance to fetch installation status for webhook when component first loads
+  useEffect(() => {
+    (async () => {
+      const hasConfiguredWebhook: boolean = await aha.account.getExtensionField(
+        IDENTIFIER,
+        "webhookConfigured"
+      );
+      setHasConfiguredWebhook(!!hasConfiguredWebhook); // coerce from null to false if the field isn't set
+    })();
+  }, []);
+
+  if (hasConfiguredWebhook === null) {
+    return <aha-spinner />;
+  }
+
+  if (pasteMode) {
+    return (
+      <EmptyStateBox>
+        <aha-flex justify-content="space-between" align-items="flex-start">
+          <div className="mb-2" style={{ fontSize: 14, fontWeight: 500 }}>
+            <aha-icon icon={ICON} class="mr-2" />
+            Paste pull request link
+          </div>
+          <aha-button size="mini" onClick={() => setPasteMode(false)}>
+            Cancel
+          </aha-button>
+        </aha-flex>
+        <input
```

Comment:
Can we auto-focus this input after we show it?

Reply:
That would be a nice touch!

Reply:
done